common/static/common/templates/discussion/thread-show.underscore

```diff
@@ -59,16 +48,15 @@
     </header>
 
     <div class="post-body"><%- body %></div>
-
-    <% if (mode == "tab" && obj.courseware_url) { %>
-        <%
-        var courseware_title_linked = interpolate(
-            '<a href="%(courseware_url)s">%(courseware_title)s</a>',
-            {courseware_url: courseware_url, courseware_title: _.escape(courseware_title)},
-            true
-        );
-        %>
-        <div class="post-context">
+    <div class="post-context">
```

Comment:
Context: This change is less drastic than github makes it seem. The "post-context" div was moved outside of the conditional logic that exposes the "related to" part of the post-context area, to account for the .group-visibility-label div moving here.